### PR TITLE
Polymer 2.0 support

### PR DESCRIPTION
**Pull request by lordoftheflies**
- Upgrading to Polymer 2.0.
- Remove moment import.

### CODE REVIEW

**Comment by saeidzebardast** [comment]
Please fix tests.

**Comment by lordoftheflies** [comment]
I will finalizing this on sept 02-03.

File: bower.json [hunk 1]
```diff
@@ -14,19 +14,22 @@
   ],
   "main": "moment-js.html",
   "license": "MIT",
-  "homepage": "https://github.com/saeidzebardast/moment-js/",
+  "homepage": "https://github.com/lordoftheflies/moment-js/",
```
Comment (masonlouchart) [inline bower.json:17]: Why the homepage URL was changed?

File: index.html [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
+  <meta http-equiv="refresh" content="0;url=demo/"/>
+  <title>moment-js</title>
   <link rel="import" href="../github-fork-ribbon/github-fork-ribbon.html">
-
 </head>
-<body unresolved>
-  <github-fork-ribbon username="saeidzebardast" repository="moment-js" position="right-bottom"></github-fork-ribbon>
-  <iron-component-page></iron-component-page>
+<body>
+<github-fork-ribbon username="lordoftheflies" repository="moment-js" position="right-bottom"></github-fork-ribbon>
```
Comment (masonlouchart) [inline index.html:10]: Why the username was changed?

File: moment-js.html [hunk 3]
```diff
@@ -1,5 +1,6 @@
-<link rel="import" href="../polymer/polymer.html">
-<link rel="import" href="./moment-import-behavior.html">
+<link rel="import" href="../polymer/polymer-element.html">
+<!--<link rel="import" href="./moment-import-behavior.html">-->
+<script src="../moment/min/moment.min.js"></script>
```
Comment (masonlouchart) [inline moment-js.html:3]: We were using a behavior to ensure not loading `moment.min.js` each time a `moment-js` tag is attached to the DOM. Have you tested it before remove the behavior?

**Comment by saeidzebardast** [comment]
Follow #36

**Pull request closed without merging**